includes/core-changes/windowsforms/3.1/remove-controls-3.1.md

```diff
@@ -53,7 +53,7 @@ Each removed control has a recommended replacement control. Refer to the followi
 | ContextMenu | ContextMenuStrip | |
 | Menu | ToolStripDropDown, ToolstripDropDownMenu | MenuItemCollection |
```

Comment:
```suggestion
| Menu | ToolStripDropDown, ToolStripDropDownMenu | MenuItemCollection |
```

Reply:
@tdykstra do you have permission to commit this?

Reply:
https://github.com/dotnet/docs/pull/16764